<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RandomValuesDialog</class>
 <widget class="QDialog" name="RandomValuesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>212</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="generateValues">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Generate Random Values</string>
   </property>
  </widget>
  <widget class="QLabel" name="generateLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>You can enter your own values. 
Will use initial values if blank.</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="minXValueLabel">
      <property name="text">
       <string>Min X Value</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="minXValueLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="maxXValueLabel">
      <property name="text">
       <string>Max X Value</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="maxXValueLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="minYValueLabel">
      <property name="text">
       <string>Min Y Value</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="minYValueLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="maxYValueLabel">
      <property name="text">
       <string>Max Y Value</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="maxYValueLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="amountLabel">
      <property name="text">
       <string>Amount</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="amountLineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
